opengeode/opengeode.py

```diff
@@ -1069,7 +1069,7 @@ def paste_symbols(self):
 
     def sdl_to_statechart(self, basic=True, view=None, ast=None):
         ''' Create a graphviz representation of the SDL model 
-            Optionally take a QGraphicsView to use as parent for modals '''
+            Optionally take a QGraphicsView to use as parent for models '''
```

Comment:
I'm not terribly confident about this. `modal` in relation to Qt could refer to a modal dialog...

Reply:
Indeed `modals` is what is meant here.